app/models/study_file.rb

```diff
@@ -24,7 +24,7 @@ class StudyFile
   # constants, used for statuses and file types
   STUDY_FILE_TYPES = ['Cluster', 'Coordinate Labels' ,'Expression Matrix', 'MM Coordinate Matrix', '10X Genes File',
                       '10X Barcodes File', 'Gene List', 'Metadata', 'Fastq', 'BAM', 'BAM Index', 'Documentation',
-                      'Other', 'Analysis Output', 'Ideogram Annotations', 'Image', 'AnnData', 'Seurat'].freeze
+                      'Other', 'Analysis Output', 'Ideogram Annotations', 'Image', 'AnnData Reference', 'AnnData Ingestible', 'Seurat'].freeze
```

Comment:
Blocker: per other comment:

```suggestion
                      'Other', 'Analysis Output', 'Ideogram Annotations', 'Image', 'AnnData', 'AnnData Reference', 'Seurat'].freeze
```